src/components/table/table.vue

```diff
@@ -309,6 +309,9 @@ export default {
     }
   },
   methods: {
+    isNil($val) {
```

Comment:
Why the dollar sign in the name? I thought for a sec that it was a Vue internal..

Lets call it `val` instead of `$val`

```suggestion
    isNil(val) {
```

Reply:
... i thought that about `val` ;)